### DISCUSSION TITLE
What is the status of the modoboa-users google group?

### DISCUSSION THREAD

**Discussion by foobarbecue** [modoboa/modoboa#3255] [Q&A]
I saw references to information available on the mailing list at https://groups.google.com/g/modoboa-users/ in the issue tracker and went to go check it out. However, when I tried to sign up I got a "An error occurred while joining the group" and when I go to the https://groups.google.com/g/modoboa-users/  it has a notice about it being blocked. What's the story? Is this list active?

**Comment by Spitfireap** [comment]
Hi, the mailing list has been removed. We use discord now :)

**Comment by foobarbecue** [comment]
How do I get to the discord server?

On 2024-05-27 08:17, Spitfireap wrote:

> Hi, the mailing list has been removed. We use discord now :)
> 
> --
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].
> You are receiving this because you authored the thread.Message ID: 
> ***@***.***>



Links:
------
[1] 
https://github.com/modoboa/modoboa/discussions/3255#discussioncomment-9571640
[2]
